includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -62,4 +92,36 @@ public function get_summary( $request ) {
 		return rest_ensure_response( $summary );
 	}
 
+	/**
+	 * Renders HTML for a print receipt
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return void
+	 */
+	public function generate_print_receipt( $request ) {
+		include_once WCPAY_ABSPATH . 'includes/in-person-payments/class-wc-payments-in-person-payments-receipt-service.php';
+
+		$payment_id     = $request->get_param( 'payment_id' );
+		$payment_intent = $this->api_client->get_intent( $payment_id );
```

Comment:
Let's pass `$request->get_param( 'payment_id' )` in here, without an extra variable introduction. Same for lines below.

Reply:
Done!